CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-# librdkafka v2.9.0
+# librdkafka v2.10.0
 
-librdkafka v2.9.0 is a feature release:
+librdkafka v2.10.0 is a feature release:
```

Comment:
Need to add KIP-848 related changes as well including regex support. Will create a new PR for that.